src/utils/AccessTokenRequestValidations.ts

```diff
@@ -0,0 +1,42 @@
+import { AccessRequestPayload } from "../type/AccessRequestPayload";
+import { AppError } from "./AppError";
+import { Constants } from "./Constants";
+import { ISessionItem } from "../models/ISessionItem";
+import { AuthSessionState } from "../models/enums/AuthSessionState";
```

Comment:
Thanks for updating that function! Think this is a left over import that's not needed anymore